spec/suites/layer/marker/MarkerSpec.js

```diff
@@ -237,17 +237,19 @@ describe("Marker", function () {
 			expect(mapSpy.called).not.to.be.ok();
 		});
 
-		it("do not catch event if it does not listen to it", function () {
+		it("do not catch event if it does not listen to it", function (done) {
 			var marker = new L.Marker([55, 37]);
 			map.addLayer(marker);
 			marker.once('mousemove', function (e) {
 				// It should be the marker coordinates
-				expect(e.latlng).to.be.nearLatLng(marker.getLatLng());
+				expect(e.latlng.equals(marker.getLatLng())).to.be.equal(true);
```

Comment:
Not really sure. We're using a custom version of `expect.js`, and the behaviour of `to.be` and `to.be.equal` and `to.eql` is *funny*. We could see if changing to a NPM package for this would make this any better.